PIL/Image.py

```diff
@@ -1959,6 +1962,25 @@ def split(self):
                 ims.append(self._new(self.im.getband(i)))
         return tuple(ims)
 
+    def getchannel(self, channel):
+        """
+        Returns an image contained single channel of the source image.
+
+        :param channel: What channel to return. Could be index
+          (0 for "R" channel of "RGB") or channel name
```

Comment:
Do we need negative indexing? (-1 for alpha channel in "RGBA" and "K" in "CMYK")